### PR TITLE
Added a lot more parameters

### PR DESCRIPTION
**Pull request by bschonec**
Added parameters that haven't yet been included.

Updated README.md.

### CODE REVIEW

File: spec/classes/init_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 describe 'nslcd' do
 
-  {'Ubuntu' => 'Debian', 'Debian' => 'Debian'}.each do |system, family|
+  {'RedHat' => 'Debian', 'Ubuntu' => 'Debian', 'Debian' => 'Debian'}.each do |system, family|
```
Comment (t0xicCode) [inline spec/classes/init_spec.rb]: Shouldn't this be `'RedHat' => 'RedHat', `?

**Comment by t0xicCode** [comment]
@bschonec Could you rebase on top of master?

**Comment by bschonec** [comment]
As soon as I figure out what and how to do that (rebase) I will.

File: spec/classes/init_spec.rb (hunk 1 above)
Comment (bschonec) [inline spec/classes/init_spec.rb]: Don't know.  I have no idea how to create SPEC files/checks.

Comment (t0xicCode) [inline spec/classes/init_spec.rb]: @werekraken added support for RedHat-based OS in PR #1. So that can help simplify your PR too :)

**Pull request closed without merging**